include/extractor/extraction_way.hpp

```diff
@@ -50,6 +65,20 @@ struct ExtractionWay
     void set_backward_mode(const TravelMode m) { backward_travel_mode = m; }
     TravelMode get_backward_mode() const { return backward_travel_mode; }
 
+    // wrappers to allow assigning nil (nullptr) to string values
+    void SetName(const char* value) { detail::maybeSetString(name, value); }
```

Comment:
Might be that this is not possible. But before lua was assigning the values itself (below you changed this to a property with set/get methods). I am asking if we could have a function of the style `SetStringValue(std::string &destination, const char* value)` instead of a dedicated one for every value. 
But that is just a proposal. We would need to call this function from `lua` which can be considered just as ugly. Feel free to stick with the solution we have here.